hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -352,6 +352,11 @@
   public static final String HDDS_TRACING_ENABLED = "hdds.tracing.enabled";
   public static final boolean HDDS_TRACING_ENABLED_DEFAULT = false;
 
+  public static final String SCM_DB_CHECKPOINT_TRANSFER_RATE_KEY =
```

Comment:
Can you please use Java based configuration API:

https://cwiki.apache.org/confluence/display/HADOOP/Java-based+configuration+API

Reply:
yup. I will upload new changes.